cypress/integration/dataEntry/dataEntry_bpartner_spec.js

```diff
@@ -1,7 +1,7 @@
-import { DataEntryTab, DataEntrySubTab } from '../../support/utils/dataEntryTab';
-import { DataEntrySection, DataEntryLine } from '../../support/utils/dataEntrySection';
+import { DataEntrySubTab, DataEntryTab } from '../../support/utils/dataEntryTab';
+import { DataEntryLine, DataEntrySection } from '../../support/utils/dataEntrySection';
 import { DataEntryField, DataEntryListValue } from '../../support/utils/dataEntryField';
-import { humanReadableNow } from "../../support/utils/utils";
+import { humanReadableNow } from '../../support/utils/utils';
 
 describe('Create bpartner with custom dataentry based tabs', function() {
   const date = humanReadableNow();
```

Comment:
Shouldn't we create a fixture for this test as well?

Reply:
I tried, but then i gave up. The test's messy and it will/would take way too much time. Plus dataentry is difficult to use/test because for addressing subtab/field/etc. we need the id of the "newly created thing".
I don't wanna do it, but if you wanna waste (a lot) of time on this, go ahead :D

Reply:
ok:))) got the message, I didn't know that:))